Templates/LinuxMainAllTestsGenerate.stencil

```diff
@@ -0,0 +1,20 @@
+// sourcery:file:Tests/LinuxMain.swift
+#if os(Linux)
+import XCTest
+
+{% for type in types.classes|based:"TestCase" %}
+{% if not type.annotations.disableTests %}extension {{ type.name }} {
```

Comment:
Add newline under `for type in types...`, and remove newline under `import XCTest`.

Thus extension won't be glued to each other

Reply:
fixed